scripts/actions/mobskills/murk.lua

```diff
@@ -14,8 +14,8 @@ end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local typeEffect = nil
-    local slowed     = xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.SLOW, 1250, 0, 60)
-    local weight     = xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.WEIGHT, 40, 0, 60)
+    local slowed     = xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.SLOW, 8500, 0, 90)
```

Comment:
Should leave a comment here about jimmy's notes with the haste preventing the slow, not 100% sure if our interactions here would work (unless you verified this works!)

Reply:
Was able to flag it to be overwritten by haste by setting subpower to 1.